renovatebot/renovate
# chore: set up Vale

## Changes

- Set up Vale for use with Gitpod
- Implement some "Plain language guidelines" in YAML for Vale
- Set up custom vocabulary (`accept.txt` and `reject.txt`) for the Renovate bot project

## Context

Helps with #14835, but doesn't close it.

## How to test

1. [Open the `chore/vale-linter` branch on my fork in Gitpod](https://gitpod.io/#https://github.com/HonkingGoose/renovate/tree/chore/vale-linter)
1. Wait for Gitpod to finish the setup steps
1. Use the terminal to run `vale docs/`
1. See Vale output with `error` and `suggestion` level warnings

Tip: you can open any branch in Gitpod by adding `https://gitpod.io/#` before the GitHub link to the branch. 😉

Tip: use `vale -h` to get a full list of options for the `vale` command.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository
